lib/bounces.js

```diff
@@ -47,6 +47,19 @@ module.exports = function (log, error) {
         // But we can measure it!
         log.increment('account.email_bounced.already_verified')
       }
+      // Because we have no session token on this endpoint, metrics context
+      // metadata was stashed against a synthesized token for the benefit of
+      // the activity events. This fake request object allows the correct
+      // metadata to be gathered when we emit the events.
+      var fakeRequestObject = {
```

Comment:
Maybe that decision to stash per-event was wrong btw. I think we made it before we brought the lifetime of the stashed data down from infinity to 30 minutes. It seems entirely reasonable now to remove the event name from the key and just store data against the token (or faked token) instead.